Parkinson/Startup/APHProfileExtender.m

```diff
@@ -33,7 +33,17 @@
  
 #import "APHProfileExtender.h"
 
-static  NSInteger  kDefaultNumberOfExtraSections =  2;
+// ***************************************************************************
+// ***************************************************************************
+// Of Note: This class is adding extra sections as of mPower 1.1 because no special
+// options were still required. In order to add special sections, set the
+// kDefaultNumberOfExtraSections to the # required and adjust the decorateCell
+// and didSelectorRowAtIndexPath delegate methods to reflect the new section(s)
+// desired.
+// ***************************************************************************
+// ***************************************************************************
+
+static  NSInteger  kDefaultNumberOfExtraSections =  0;
```

Comment:
Would be good to know (and note in the comments) if there’s a reason it should be 2 for one added section—that’s kind of not intuitive…

> On Aug 20, 2015, at 5:58 PM, Eric Sieg <notifications@github.com> wrote:
> 
> In Parkinson/Startup/APHProfileExtender.m <https://github.com/Sage-Bionetworks/mPower/pull/54#discussion_r37597985>:
> 
> > @@ -33,7 +33,17 @@
> >   
> >  #import "APHProfileExtender.h"
> >  
> > -static  NSInteger  kDefaultNumberOfExtraSections =  2;
> > +// ***************************************************************************
> > +// ***************************************************************************
> > +// Of Note: This class is adding extra sections as of mPower 1.1 because no special
> > +// options were still required. In order to add special sections, set the
> > +// kDefaultNumberOfExtraSections to the # required and adjust the decorateCell
> > +// and didSelectorRowAtIndexPath delegate methods to reflect the new section(s)
> > +// desired.
> > +// ***************************************************************************
> > +// ***************************************************************************
> > +
> > +static  NSInteger  kDefaultNumberOfExtraSections =  0;
> I think its related to the spacing, but I didn't investigate that deeply other than making sure the sections after matched what was there before other than the deprecated medication tracker.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Sage-Bionetworks/mPower/pull/54/files#r37597985>.
>